pkg/exchange/okex/stream.go

```diff
@@ -79,17 +79,18 @@ func NewStream(client *okexapi.RestClient, balanceProvider types.ExchangeAccount
 
 func (s *Stream) UseFutures() {
 	s.isFutures = true
```

Comment:
you can get the FuturesSettings from the exchange instance

Reply:
got it, I'll give it a try